wallbox_client.py

```diff
@@ -453,10 +453,14 @@ def handle_charger_state_change(self, entity, attribute, old, new, kwargs):
         # Goes to this status when disconnected
         if new_charger_state == self.registers["disconnected_state"]:
             self.log("Charger state has changed to 'Disconnected'")
-            # Send this to FM?
             # Cancel current scheduling timers
             self.cancel_charging_timers()
 
+            # When the car disconnects we want a chargemode Max Boost Now to end.
```

Comment:
I found the user notification in line 463 a lot more insightful than this line.

Reply:
Changed to:
Setting charge_mode set to automatic (was Max boost Now) as car is disconnected.